Integrations/integration-ArcherRSA.yml

```diff
@@ -2426,6 +2471,7 @@ script:
 
         if (params.useRest){
             // in case that the user wants to use the REST API
+            fieldsToValue = getIDsOfMultiselectFields(fieldsToValue, createRecordMapping);
```

Comment:
what is this function getIDsOfMultiselectFields ? can't find it in the code

Reply:
fixed